app/assets/javascripts/host_edit_interfaces.js

```diff
@@ -29,6 +29,7 @@ function show_interface_modal(modal_content) {
   modal_window.modal({'show': true});
 
   modal_window.find('a[rel="popover-modal"]').popover({html: true});
+  $('fieldset.compute_attributes select').select2();
```

Comment:
Yeah, I'm inclined to agree, I dislike the inconsistency of it being only on a few select boxes.  The only nitpick is that it's slightly larger and possibly a fraction slower compared to a regular select, so I don't know how it'll make the app feel if we replaced them all.